services/movie_session.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime, timedelta
+
+from django.db.models import QuerySet
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(
+        movie_show_time: str,
```

Comment:
Consider validating the `movie_show_time` format to ensure it is a valid datetime string before creating a `MovieSession`. This will help prevent runtime errors if an invalid date format is provided.